index.rst

```diff
@@ -437,11 +447,124 @@ in some contexts where materializing the set would be expensive. The
 'DataRefSet' is usually generated by listing existing datasets of a particular
 dataset type, but its component 'DataRef's can be used with other dataset types.
 
+
+DAX
+===
+
+Overview - What is DAX?
+-----------------------
+
+DAX is a collection of Data Access Interfaces implemented as REST APIs.
+Currently, there are three core APIs: dbserv, metaserv, and imgserv. The are
+all implemented in python using the `Flask framework
+<http://flask.pocoo.org/>`_. The collection of these APIs referred to
+as webserv.
```

Comment:
could be more clear? "The collection of these 3 APIs is loosely referred to as webserv." ?